ddsrecordreplay.repos

```diff
@@ -10,7 +10,7 @@ repositories:
     fastdds:
         type: git
         url: https://github.com/eProsima/Fast-DDS.git
-        version: bugfix/complex-dynamic-types
+        version: master
```

Comment:
seems that `bugfix/complex-dynamic-types` includes the fixes that are not merged into master yet, but according to the experimental verification https://github.com/eProsima/DDS-Record-Replay/issues/106#issuecomment-1876117990, this seems to be working fine.